pyrevitlib/pyrevit/compat.py

```diff
@@ -32,25 +36,6 @@
     import urllib
     from urllib.parse import urlparse
 
-
```

Comment:
Solve the merge conflict, we extracted the revit version check function to be reusable (and it's needed a few lines below)

Reply:
I've resolved the conflict.